tf-modules/distribution/main.tf

```diff
@@ -24,12 +24,13 @@ module "tea_map_cache" {
 }
 
 data "aws_lambda_invocation" "tea_map_cache" {
-  depends_on    = [module.tea_map_cache.lambda_function_name]
-  function_name = module.tea_map_cache.lambda_function_name
-  input = jsonencode({
-    bucketList = local.tea_buckets,
-    s3Bucket = var.system_bucket
-    s3Key    = "${var.prefix}/distribution_bucket_map.json"
+  function_name         = module.tea_map_cache.lambda_function_name
+  input                 = jsonencode({
+    bucketList          = local.tea_buckets,
+    s3Bucket            = var.system_bucket
+    s3Key               = "${var.prefix}/distribution_bucket_map.json"
+    teaEndpoint         =  var.tea_internal_api_endpoint
```

Comment:
Is this input necessary? I don't see a `teaEndpoint` property used by the tea-map-cache lambda

Reply:
No, it's a leftover from attempting to rotate the endpoint.   
  Good catch. 3871f395fef6bfca61e64823533c3b13c3748228